cmd/curator/curator.go

```diff
@@ -225,17 +225,34 @@ func curatorRun(config *rest.Config, client *clientv1.Client, clusterName string
 				jobChoice,
 				v1.ConditionTrue,
 				err.Error()))
+			klog.Error(err.Error())
+			panic(err)
+		}
+	}
+
+	if jobChoice == "prehook-ansiblejob" {
```

Comment:
You don't want to just do an OR for posthook-ansiblejob or prehook-ansiblejob so we don't get a code smell?

Reply:
i can combine these 2 if for pre and post